src/chain_spec.rs

```diff
@@ -1264,31 +1264,5 @@ fn asset_registry_assets() -> Vec<(CurrencyId, Vec<u8>)> {
 			}
 			.encode(),
 		),
-		(
```

Comment:
@wischli the block above is registering AUSD, this deleted block was a duplicate. I don't know how we merged this in 🤔

Reply:
ahah no worries! glad it's clear!